rasa/core/domain.py

```diff
@@ -106,7 +107,7 @@ def get_duplicate_exception_message(
                 message += (
                     "Duplicate {0} in domain. "
                     "These {0} occur more than once in "
-                    "the domain: {1}".format(name, ", ".join(d))
+                    "the domain: '{1}'".format(name, ", ".join(d))
```

Comment:
Maybe these quotes should also go in the join function so that they go around each slot/entity/intent (not 100 percent sure about suggestion, test it out
```suggestion
                    "the domain: '{1}'".format(name, "\', \'".join(d))
```